pod/perldata.pod

```diff
@@ -468,14 +470,20 @@ characters such as newline, tab, etc., as well as some more exotic
 forms.  See L<perlop/"Quote and Quote-like Operators"> for a list.
 X<string, literal>
 
-Hexadecimal, octal, or binary, representations in string literals
+Hexadecimal, octal, or binary representations in string literals
 (e.g. '0xff') are not automatically converted to their integer
 representation.  The hex() and oct() functions make these conversions
 for you.  See L<perlfunc/hex> and L<perlfunc/oct> for more details.
 
 Hexadecimal floating point can start just like a hexadecimal literal,
 and it can be followed by an optional fractional hexadecimal part,
-but it must be followed by C<p>, an optional sign, and a power of two.
+but it must be followed by C<p>, an optional sign, and an exponent.  The
+digits in the exponent are interpreted as being base 10 (digits 0-9 are
+accepted), but the actual value is two raised to that power.  Thus
+
+   5e2 == 5 * 10-squared == 500
+ 0x5p2 == 5 *  2-squared ==  20
```

Comment:
An example with a 2 digit exponent for one of the bitwise bases (hex etc) would better illustrate the base-10-ness of the exponent number vs the base-2-ness of the power.